src/v2/components/ChannelContents/index.tsx

```diff
@@ -106,21 +104,13 @@ const ChannelContents: React.FC<Props> = WithIsSpiderRequesting<ExtendedProps>(
     )
 
     usePusher({
-      channel: pusherChannel,
+      channelId: channel.id,
+      skip: !isSpiderRequesting,
```

Comment:
And this should actually not be called if the person cannot add to the channel as well:

```js
const shouldSubscribeToPusher =
           !isSpiderRequesting && channel.can.add_to
```

Reply:
fixed and simplified the boolean logic here https://github.com/aredotna/ervell/pull/2554/commits/9ac0b5eb0700f3cb859f6bec31525d7d38438f9c